static/sass/_patterns_footer.scss

```diff
@@ -0,0 +1,234 @@
+      + .second-level-nav {
+        display: none;
+
+        @media only screen and (min-width: $breakpoint-medium) {
+          display: block;
+        }
+      }
+
+      &.active {
+        background-image: url("#{$assets-path}43e2b367-arrow_up_9fa097.png");
+
+        @media only screen and (min-width: $breakpoint-medium) {
+          background: none;
+        }
+
+        + .second-level-nav {
+          display: block;
+        }
+      }
+    }
+
+    &__container {
+      @media only screen and (max-width: $breakpoint-medium - 1px) {
+        padding-left: 0;
+        padding-right: 0;
+      }
+    }
+
+    &__link {
+      &:hover {
+        color: $color-dark;
+      }
+    }
+
+    .link-to-top {
+      @include footer-title;
+      background-image: url("#{$assets-path}e4a5a8c0-picto-upload-warmgrey.svg");
+      background-position: 10px center;
+      background-repeat: no-repeat;
+      background-size: 14px 14px;
+      border-bottom: 1px solid $color-mid-dark;
+      display: block;
+      left: auto;
+      margin-bottom: -1px;
+      padding: 10px 0 10px 30px;
+      position: relative;
+      z-index: 2;
+
+      @media only screen and (min-width: $breakpoint-medium) {
+        display: none;
+      }
+
+      a {
+        color: $color-mid-dark;
+      }
+    }
+
+    // second level nav - HTML for this can't be changed as it's used in the main nav also
+    .second-level-nav {
+      margin: 0;
+      padding: 0;
+
+      li {
+        margin-bottom: 0;
+
+        a {
+          border-top: 1px solid $color-mid-light;
+          color: $color-dark;
+          display: block;
+          font-size: .867rem;
+          padding: 10px 10px 10px 20px;
+
+          @media only screen and (min-width: $breakpoint-medium) {
+            border: 0;
+            font-size: .8125rem;
+            padding: 0 0 5px;
+          }
+        }
+      }
+    }
+  }
+
+  .p-footer--secondary {
+    padding: 20px 10px 0;
+
+    &__content {
+      margin-bottom: .3rem;
+    }
+
+    @media only screen and (min-width: $breakpoint-medium) {
+      padding: 0;
+    }
+
+    a:hover {
+      color: currentColor;
+    }
+
+    .button--primary {
+      background-color: $color-x-light;
+      border: 1px solid $color-mid-light;
+      color: $color-dark;
+      padding: 6px 18px;
+
+      &:hover {
+        background-color: $color-light;
+      }
+    }
+
+    .p-inline-list__item {
+      display: block;
+
+      @media only screen and (min-width: $breakpoint-medium) {
+        display: inline-block;
+      }
+    }
+  }
+
+  // social icons in footer
+  .p-inline-list-icons {
+    @media only screen and (min-width: $breakpoint-medium) {
+      float: right;
+      padding-left: 0;
+    }
+
+    @media only screen and (min-width: $breakpoint-large) {
+      margin-top: 10px;
+    }
+  }
+
+  .p-inline-list,
+  .p-inline-list--small,
+  .p-inline-list--middot-small {
+    margin-left: 0;
+    margin-top: 0;
+    padding-left: 0;
```

Comment:
These patterns already have padding and margin left set to 0 in the pattern themselfs.